### PR TITLE
Minor include directive fixes.

### PR DESCRIPTION
**Pull request by chsigg**

### CODE REVIEW

**Comment by kentrussell** [comment]
I've brought these patches internally for testing and review. I am not sure if it will make into the 2.8 release, but we'll see what we can do. Thanks for bringing these to our attention, and for helping to improve the ROCR project!

**Comment by chsigg** [comment]
Great, thanks a lot! For now I just patch the #include directives when
importing the repo, so this is not blocking us.
I also simply change all 'throws' to do nothing, which might be a problem.

File: src/core/inc/sdma_registers.h [hunk 1]
```diff
@@ -43,6 +43,9 @@
 #ifndef HSA_RUNTIME_CORE_INC_SDMA_REGISTERS_H_
 #define HSA_RUNTIME_CORE_INC_SDMA_REGISTERS_H_
 
+#include <cstddef>
```
[the remark this answers is not in the document]
Comment (chsigg) [reply src/core/inc/sdma_registers.h]: Yes, sorry. It should have been stddef.h (for size_t) and stdint.h (for uint16_t).

**Comment by skeelyamd** [comment]
These patches are in the pipe for being in the binary releases.  I'll merge them when I can be sure that they will be in the build for the next binary release.

**Comment by skeelyamd** [comment]
This will be in the 2.9 packages.

**Pull request merged**